pipeline/transformer/transformer_tei/src/main/scala/weco/pipeline/transformer/tei/transformers/TeiContributors.scala

```diff
@@ -0,0 +1,276 @@
+    * within a handNote tag with attribute "scribe" as in this example
+    * <physDesc>
+    *     <handDesc>
+    *         <handNote scope="minor" scribe="Scribe_A"> <locus target="#Wellcome_Batak_36801_1 #Wellcome_Batak_36801_2" >a 2-62, b 2-7; b 8-22</locus> Gorak-gorahan, etc. Southern form of ta.</handNote>
+    *         <handNote scope="minor" scribe="Scribe_B"> <locus target="#Wellcome_Batak_36801_3">b 23-35</locus>Another hand, which uses the northern ta.</handNote>
+    *         <handNote scope="minor" scribe="Scribe_C"> <locus target="#Wellcome_Batak_36801_9">b 44-45</locus>Another hand; careless writing, at first with northern ta, later southern ta.</handNote>
+    *         <handNote scope="minor" scribe="Scribe_D"> <locus target="#Wellcome_Batak_36801_10">b 45-56</locus>A different handwriting, using the southern form of ta but not the peculiar form of ya which is found in the text written by <persName>Datu Poduwon</persName>.</handNote>
+    *     </handDesc>
+    * </physDesc>
+    *
+    * or within a persName tag with role=scr inside the handNote tag
+    * <physDesc>
+    *    <handDesc>
+    *        <handNote scope="sole">
+    *            <persName role="scr">Mahādeva Pāṇḍe</persName>
+    *        </handNote>
+    *    </handDesc>
+    *</physDesc>
+    * If the scribe refers to a part or item of the manuscript, the handNote tag will have a locus node with the item or part ids that it refers to
+    *
+    */
+  def scribes(
+    xml: Elem,
+    workId: String): Result[Map[String, List[Contributor[Unminted]]]] =
+    (xml \\ "physDesc" \ "handDesc" \ "handNote").foldLeft(
+      Right(Map.empty): Result[Map[String, List[Contributor[Unminted]]]]
+    ) {
+      case (Left(err), _) => Left(err)
+      case (Right(scribesMap), node) =>
+        for {
+          contributor <- getScribeContributor(node)
+        } yield mapContributorToWorkId(workId, node, contributor, scribesMap)
+    }
+
+  /**
+    * Author nodes can be in 2 forms:
+    * <msItem xml:id="MS_Arabic_1-item1">
+    *  <author key="person_97166546">
+    *    <persName xml:lang="en">Avicenna, d. 980-1037
+    *    </persName>
+    *    <persName xml:lang="ar" type="original">ابو على الحسين ابن عبد الله ابن
+    *    سينا</persName>
+    *  </author>
+    *  or
+    * <msItem n="1" xml:id="MS_MSL_114_1">
+    *  <author key="person_84812936">Paul of Aegina</author>
+    *  So we must check for the existence of the internal persName nodes to decide
+    *  where to get the label and id from
+    */
+  private def getLabelAndId(n: Node) = (n \ "persName").toList match {
+    case Nil            => getFromAuthorNode(n)
+    case List(persNode) => getFromPersNode(n, persNode)
+    case list           => getFromOriginalPersNode(n, list)
+  }
+
+  private def getScribeContributor(n: Node) =
+    for {
+      maybeLabel <- parseScribeLabel(n)
+      contributor <- maybeLabel.map { label =>
+        createContributor(label, ContributionRole("scribe"))
```

Comment:
Because we're not doing anything fancy with IDs or `isFihrist` here, let's inline the contributor to make this a tad easier to follow:

```suggestion
        Contributor(Person(label), roles = List(ContributionRole("scribe")))
```